config/locales/controllers/notes_controller/en.yml

```diff
@@ -16,3 +16,4 @@ en:
     highly_liked_research_notes: "Highly liked research notes"
     popular_research_notes: "Popular research notes"
     more_than_one_contributor: "You cannot delete a wiki page once someone else has begun to contribute to it"
+    saved_as_draft: "Your contribution has been saved as draft and will be visisble to you only"
```

Comment:
A very tiny change needed with the word `visible` here.

Reply:
thanks @seafr and changes done